src/test/java/com/crud/tasks/domain/CreatedTrelloCardDtoTestSuite.java

```diff
@@ -0,0 +1,21 @@
+package com.crud.tasks.domain;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest
+class CreatedTrelloCardDtoTestSuite {
```

Comment:
Co to jest??? Jeszcze caly kontekst springa wlaczasz, aby sprawdzic, czy java dziala